frontend/packages/pipelines-plugin/src/components/pipelineruns/detail-page-tabs/__tests__/PipelineRunVisualization.spec.tsx

```diff
@@ -15,8 +16,15 @@ const pipelineRun = pipelineData.pipelineRuns[DataState.SUCCESS];
 describe('Pipeline Run Visualization', () => {
   type PipelineRunVisualizationProps = React.ComponentProps<typeof PipelineRunVisualization>;
   let wrapper: ShallowWrapper<PipelineRunVisualizationProps>;
+  let useEffect: jest.SpyInstance;
+  const mockUSeEffect = () => {
```

Comment:
```suggestion
  const mockUseEffect = () => {
```

Reply:
You are also not mocking `useEffect`, you're spying on its execution. If you were mocking it, k8sGet and the get utility wouldn't get called because you would have replaced it's usage.

Reply:
I chose the name since we are mocking its implementation.